packages/primitives/src/Input.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import {
+  ComponentPropsWithRef,
+  HTMLAttributes,
+  createContext,
+  forwardRef,
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+} from "react";
+import { css, cx } from "@ndla/styled-system/css";
+import { styled } from "@ndla/styled-system/jsx";
+import { composeRefs } from "@ndla/util";
+
+interface InputContextType {}
+
+const InputContext = createContext<InputContextType | undefined>(undefined);
+
+const inputCss = css.raw({
+  outline: "1px solid",
+  outlineColor: "brand.grey",
+  background: "brand.greyLightest",
+  borderRadius: "small",
```

Comment:
Er det verktøystøtte for dette? Ellers ser det litt ut som store muligheter for feil. Hadde forventa import av brand og så bruk av brand.grey osv.

Reply:
Du får autocomplete via typescript. Kan legge inn et valg som gir deg feilmelding dersom du bruker en ugyldig token.